### DISCUSSION TITLE
CheatSheet

### DISCUSSION THREAD

**Discussion by panbhatt** [TypedProject/ts-express-decorators#2883] [General]
Is there anyone who has worked on creating a cheat sheet for this framework. I would suggest @Romakita, Please look into this direction and try to create a 3-4 page cheat sheet, as this would be very helpful for us and remember all the nitty gritty details in our mind all the time. I love this framework.

**Comment by Romakita** [comment]
It’s a great idea. We can start this work on this discussion by adding small mandatory stuff for the cheat sheet ;)
